docs/products/postgresql/concepts/pg-audit-logging.md

```diff
@@ -169,13 +169,14 @@ For information on all the configuration parameters, preview
 
 ### Collecting and visualizing logs
 
-You can
-[access your collected audit logs](/docs/products/postgresql/howto/use-pg-audit-logging)
-either directly in the log output of your Aiven for PostgreSQL service or by integrating
-with another service that allows monitoring and analyzing logs, such as Aiven for
-OpenSearch®. To
-[visualize your audit logs](/docs/products/postgresql/howto/use-pg-audit-logging), you can
-use [OpenSearch Dashboards](/docs/products/opensearch/dashboards).
+[Access and visualize collected audit logs](/docs/products/postgresql/howto/use-pg-audit-logging)
+using one of the supported methods:
+
+| Methodology | Accessing `pgaudit` logs | Visualizing `pgaudit` logs |
+| :--- | :--- | :--- |
+| Aiven-native integration | [Aiven for OpenSearch®](/docs/products/opensearch/)| [OpenSearch Dashboards](/docs/products/opensearch/dashboards) |
```

Comment:
Nit: the style guide suggests avoiding the term "native": docs/platform/concepts/service-power-cycle